llvm/lib/Target/PICMid/GISel/PICMidLegalizerInfo.cpp

```diff
@@ -89,3 +135,97 @@ PICMidLegalizerInfo::PICMidLegalizerInfo() {
 
   // Miscellaneous
```

Comment:
You need to compute the Legalizer table, right? If so, pls include the fix as outlined here: https://github.com/llvm-pic/llvm-pic/issues/68#issuecomment-2104545572.